distribution/windows/install.nsi

```diff
@@ -104,31 +104,31 @@ ManifestDPIAware true
 
 ; Set languages (first is default language). Other languages are sorted by ISO code.
```

Comment:
Could you please add a comment here explaining the reason for having them commented out?